optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/ranking/SingleBenchmarkRankingComparator.java

```diff
@@ -31,8 +31,8 @@
     public int compare(SingleBenchmarkResult a, SingleBenchmarkResult b) {
         return new CompareToBuilder()
                 .append(b.isFailure(), a.isFailure()) // Reverse, less is better (redundant: failed benchmarks don't get ranked at all)
-                .append(b.getUninitializedVariableCount(), a.getUninitializedVariableCount()) // Reverse, less is better
-                .append(a.getScore(), b.getScore(), resilientScoreComparator)
+                .append(b.getAverageUninitializedVariableCount(), a.getAverageUninitializedVariableCount()) // Reverse, less is better
+                .append(a.getAverageScore(), b.getAverageScore(), resilientScoreComparator)
```

Comment:
The question here is, if we want to do this change (readability vs. backwards compatibility). We could keep using `.getScore()` and `.getUninitializedVariableCount()` and just internally flip the switch to mirror `.getAverageScore()` and `.getAvg...Count()`. This would mean reverting the changes I made all across the benchmarker code base.

See `SingleBenchmarkResult#getScore()`.  WDYT?